README.md

```diff
@@ -1,2 +1,4 @@
-# carpet-autoCraftingTable
-autoCraftingTable extension for carpet-mod. Implementation based on Skyrising's implementation for autoCraftingTable for QuickCarpet - implemnted here as an extension for fabric-carpet mod.
+# Carpet AutoCraftingTable - Unofficial Fork
+
+### IF YOU WANT THE MOD, GO HERE: https://github.com/gnembon/carpet-autoCraftingTable
```

Comment:
que?

Reply:
Sorry, I made the same mess up on multiple mods... I forgot about the pull request and randomly edited the main branch. Imma undo the commit and temporarily move it to another branch (on my fork) which is unrelated to this pr. I will comment here when I'm done.

Reply:
Fixed it now. Sorry....